src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  width: 726px;
+  padding: 30px 10px;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: $day-size;
+    height: $day-size;
+    font-size: $font-size;
+    background-color: $bc-day;
+    border: 1px solid $border;
+    transition: transform 0.3s linear, background-color 0.3s linear;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+      background-color: $day-hover;
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+      }
+    }
+  }
+
+  @each $day, $margin in $days {
+    &--start-#{$day} .calendar__day:first-child {
```

Comment:
#{$day} its the value of modifier so you should follow BEM naming convention:
block-name--modifier-name--modifier-value

Reply:
done